main.js

```diff
@@ -1 +1,47 @@
 import customers from './customers.js'
+// console.log(customers)
+
+let person = [{
+  thumbNail: "https://randomuser.me/api/portraits/thumb/women/89.jpg",
+  name: "Pauline Hughes",
+  email: "pauline.hughes@example.com",
+  address: "3747 Elgin St, Roseville, IN 93281"
+}]
+
+function listCustomers(personArray) {
+  let customerList = document.querySelector('#customer-list');
+  console.log(customerList)
+
+  for (let customer of customers) {
```

Comment:
Good start. You probably want to make this loop over `personArray`, not specifically `customers`.